### ISSUE TITLE
pretrained ms_rcnn model

### ISSUE THREAD

**Issue by fanyix** [zjhuang22/maskscoring_rcnn#21]
Thanks for the great work! 
It seems the link provided in README is for ImageNet pretrained models? Could you please provide the R-50/R-101 MS-RCNN model that is used to produce your results in the paper?

**Comment by Csysmile** [comment]
Sorry, I have a same question, how to find the R-50/R-101 MS-RCNN model?

**Comment by maomaochongchh** [comment]
@fanyix @Csysmile Hello, are you meaning that the pkl files the author provides in readme can not directly used to test? Have you run test_net.py and got similar results with the paper?

**Comment by Csysmile** [comment]
Thanks for your reply.
As readme says, the pretrained models will be downloaded when running the program, but when I run it，it can not be downloaded Automatically. I've found the pretrained models in other way.
But now I have another problem , my data have been convered to coco format and I've checked it. However after a few minutes of training the errors will be reported as "The shape of the mask[26188] at index 0 does not match the shape of the indexed tensor [0] at index 0".
I put my data in maskrcnn-benchmark as you mentioned just now, it runs normally. I'm very confused and hope you can give me some advice.

**Comment by Csysmile** [comment]
@maomaochongchh Yes , the the pkl can not be download,  I found the R-50 in other way. However I have another problem now. After a few minutes of training, the error will be reported as "the shape pf the mask[261888] at index 0 does not match the indexed tensor [0] at index 0". l put my data in maskrcnn-benchmark as the author mentions in the end and it works normally. Have you meet a similar problem?

**Comment by maomaochongchh** [comment]
sorry, I have not. When I noticed that it needs 15 days to train it with 4 gpus, I stopped it. I just want to run the test_net.py now. If you run it successfully, please tell me, thank you, and if I meet the similar problem later, I will tell you. But I suggest that you can try debug and print the value.

**Comment by Csysmile** [comment]
@maomaochongchh I downloaded it at https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md

**Comment by YubinXie** [comment]
+1
Could you please provide the R-50/R-101 MS-RCNN model that is used to produce your results in the paper?

**Comment by zjhuang22** [comment]
I've put the trained model in the corresponding link.

**Comment by XiaoLaoDi** [comment]
@fanyix @Csysmile https://github.com/XiaoLaoDi/maskscoring_rcnn here I provide the demo.py and the pretrained models on coco2017, but i am still not sure whether it matches the results of the paper, hoping can help you.
